### PR TITLE
French translation of the PHP 8 announcement page

### PR DESCRIPTION
**Pull request by Girgias**
Have done this in a hurry so a quick review would be appreciated. :)

### CODE REVIEW

**Comment by pronskiy** [comment]
@Girgias  need to add French to `$LANGUAGES` in include/layout.inc.
```diff
    $LANGUAGES = array(
        'en'    => 'English',
        'pt_BR' => 'Brazilian Portuguese',
+        'fr' => 'French',
        'de'    => 'German',
        'ru'    => 'Russian',
    );
```

File: releases/8.0/en.php [hunk 1]
```diff
@@ -123,7 +123,7 @@ class User
   <div class="php8-compare">
     <h2 class="php8-h2" id="constructor-property-promotion">
       Constructor property promotion
-      <a class="php8-rfc" href="https://wiki.php.net/rfc/constructor_promotion">RFC</a>
+      <a class="php8-rfc" href="https://wiki.php.net/rfc/constructor_promotion">RFC</a> <a class="php8-rfc" href="/manual/fr/language.oop5.decon.php#language.oop5.decon.constructor.promotion">Doc</a>
```
Comment (azjezz) [inline releases/8.0/en.php:126]:
```suggestion
      <a class="php8-rfc" href="https://wiki.php.net/rfc/constructor_promotion">RFC</a> <a class="php8-rfc" href="/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion">Doc</a>
```

File: releases/8.0/en.php [hunk 2]
```diff
@@ -172,7 +172,7 @@ public function __construct(
   <div class="php8-compare">
     <h2 class="php8-h2" id="union-types">
       Union types
-      <a class="php8-rfc" href="https://wiki.php.net/rfc/union_types_v2">RFC</a> <a class="php8-rfc" href="/manual/en/language.types.declarations.php#language.types.declarations.union">Doc</a>
+      <a class="php8-rfc" href="https://wiki.php.net/rfc/union_types_v2">RFC</a> <a class="php8-rfc" href="/manual/fr/language.types.declarations.php#language.types.declarations.union">Doc</a>
```
Comment (azjezz) [inline releases/8.0/en.php:175]:
```suggestion
      <a class="php8-rfc" href="https://wiki.php.net/rfc/union_types_v2">RFC</a> <a class="php8-rfc" href="/manual/en/language.types.declarations.php#language.types.declarations.union">Doc</a>
```

File: releases/8.0/en.php [hunk 3]
```diff
@@ -220,7 +220,7 @@ public function __construct(
   <div class="php8-compare">
     <h2 class="php8-h2" id="match-expression">
       Match expression
-      <a class="php8-rfc" href="https://wiki.php.net/rfc/match_expression_v2">RFC</a>
+      <a class="php8-rfc" href="https://wiki.php.net/rfc/match_expression_v2">RFC</a> <a class="php8-rfc" href="/manual/fr/control-structures.match.php">Doc</a>
```
Comment (azjezz) [inline releases/8.0/en.php:223]:
```suggestion
      <a class="php8-rfc" href="https://wiki.php.net/rfc/match_expression_v2">RFC</a> <a class="php8-rfc" href="/manual/en/control-structures.match.php">Doc</a>
```

File: releases/8.0/fr.php [hunk 4]
```diff
@@ -0,0 +1,523 @@
...
+    </div>
+    <div class="php8-compare__content">
+      <p>Moins de code redondant pour définir et initialiser les propriétés.</p>
+    </div>
+  </div>
+
+  <div class="php8-compare">
+    <h2 class="php8-h2" id="union-types">
+      Types d'union
+      <a class="php8-rfc" href="https://wiki.php.net/rfc/union_types_v2">RFC</a> <a class="php8-rfc" href="/manual/en/language.types.declarations.php#language.types.declarations.union">Doc</a>
```
Comment (azjezz) [inline releases/8.0/fr.php:177]:
```suggestion
      <a class="php8-rfc" href="https://wiki.php.net/rfc/union_types_v2">RFC</a> <a class="php8-rfc" href="/manual/fr/language.types.declarations.php#language.types.declarations.union">Doc</a>
```

**Review by azjezz — APPROVED** [review]
👍 just a few issues

**Comment by Girgias** [comment]
Had them fixed locally, but as I can't yet push to the git repo due to karma propagation I'll force push it :)

**Comment by Girgias** [comment]
*just when I started complaning* merged as f2407335ed9f1156a8428429342211db98bb4311

**Pull request closed without merging**